### PR TITLE
Add sonom.json

### PR DESCRIPTION
**Pull request by getSono**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [X] The file is in the `domains` folder and is in the JSON format.
- [X] You have completed your website. 
- [X] The website is reachable.  
- [  ] You're not using Vercel or Netlify.  
- [X] The CNAME record doesn't contain `https://` or `/`.  
- [X] There is sufficient information at the `owner` field.

### CODE REVIEW

File: package-lock.json
Comment (CuteDog5695) [inline package-lock.json:1]: delete this file since it is not supposed to be here

File: domains/sonom.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "username": "getSono",
+    "email": "julianwolf2013@outlook.de",
+    "discord": "1222250424309121145",
```
Comment (CuteDog5695) [inline domains/sonom.json:4]: these need to be in the owner structure (check here https://www.is-a.dev/docs/domain_structure/)

File: domains/sono.json [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
-           "email": "julianwolf2013@outlook.de",
-           "discord": "1222250424309121145"
-        },
-    
-        "record": {
-            "CNAME": "getsono.github.io"
-        }
+    "username": "getSono",
+    "email": "julianwolf2013@outlook.de",
+    "discord": "1222250424309121145",
```
Comment (CuteDog5695) [inline domains/sono.json:4]: these need to be in the owner structure (check here https://www.is-a.dev/docs/domain_structure/)

File: domains/sono.json [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
+    "record": {
+        "CNAME": "getsono.github.io",
+        "MX": [
+            "mx1.improvmx.com",
+            "mx2.improvmx.com"
+        ],
+        "TXT": "v=spf1 include:spf.improvmx.com ~all"
```
Comment (CuteDog5695) [inline domains/sono.json:11]: you cant use CNAME with any other records

**Review by iostpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**